src/words/FrameLoop.java

```diff
@@ -87,8 +88,17 @@ private boolean executeSingleFrame() {
 		}
 
 		// Phase 3: Listener Evaluation
-		for (WordsEventListener eventListener : environment.getEventListeners()) {
-			eventListener.execute();
+		for (Iterator<WordsEventListener> iterator = environment.getEventListeners().iterator(); iterator.hasNext();) {
```

Comment:
Could we use the iterator shorthand like this?  Is there a reason to prefer the more verbose form?

for (WordsEventListener : evenListener : environment.getEventListeners()) {

}